src/renderer/src/app.tsx

```diff
@@ -76,6 +82,29 @@ declare module '@tanstack/react-router' {
 	}
 }
 
+let sentryEnvironment: SentryEnvironment = 'development'
+
+if (__APP_TYPE__ === 'release-candidate') {
+	sentryEnvironment = 'qa'
+} else if (__APP_TYPE__ === 'production') {
+	sentryEnvironment = 'production'
+}
+
+const sentryUserId = window.runtime.getSentryUserId()
+
+initSentryElectron(
+	{
+		dsn: 'https://f7336c12cc39fb0367886e31036a6cd7@o4507148235702272.ingest.us.sentry.io/4509803831820288',
+		// TODO: Enable tracing based on user consent in production
+		tracesSampleRate: sentryEnvironment === 'production' ? 0 : 1.0,
+		integrations: [tanstackRouterBrowserTracingIntegration(router)],
+		environment: sentryEnvironment,
+		debug: sentryEnvironment === 'development',
+		initialScope: { user: { id: sentryUserId } },
```

Comment:
would be helpful to have some confirmation that this configured somewhat correctly